### DISCUSSION TITLE
Show the “Label” field for type Entry

### DISCUSSION THREAD

**Discussion by dolphiq-allard** [craftcms/cms#16338] [Ideas]
Hey y'all,

Recently, the Craft CMS team added the 'Show the “Label” field' option to hyperlinks. It would be great if this feature found its wat to the Entry type as well. This way building navigation out of the box would be a lot easier. Furthermore this value can be used on buttons or links etc...

Looking forward to your reactions

Cheers,
Allard

**Comment by brandonkelly** [comment]
I’m not sure I’m following you. Which entry type label do you want to be removed?

**Reply by dolphiq-allard** [reply]
Hi Brandon,

First off, thanks for your quick reply. Second, I should have done some research. The feature I was requesting is already smartly combined in the Hyperlink field, so apologies for bothering you with this one. I will close this discussion as I should have seen this.

Thanks again.

**Comment by dolphiq-allard** [comment]
Functionality already available in a different field type
